@here/olp-sdk-dataservice-read/lib/client/QuadKeyPartitionsRequest.ts

```diff
@@ -29,6 +29,7 @@ export class QuadKeyPartitionsRequest {
     private quadKey?: QuadKey;
     private depth?: QuadTreeIndexDepth;
     private billingTag?: string;
+    private additionalFields?: Array<"dataSize" | "checksum" | "compressedDataSize" | "crc">;
```

Comment:
Please add to the index.ts in service-api type and use it 
export type AdditionalFileds = Array<"dataSize" | "checksum" | "compressedDataSize" | "crc">;

Reply:
Added to the index.ts in dataservice-api